.github/workflows/version-bump.yml

```
@@ -0,0 +1,150 @@
+name: Version Bump
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)